payments/templates/payment-history.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'shop/base.html' %}
+{% load crispy_forms_tags %}
+{% block content %}    
+    <div class="content-section">
+        <form method="POST">
+            {% csrf_token %}
+            <fieldset class="form-group">
+                <legend class="border-bottom mb-4">Payment history</legend>
+                {{ form|crispy }}
+            </fieldset>
+                  <table class="demo">
+            <thead>
+            <tr>
+                <th>Payment date</th>
+                <th>Amount</th>
+                <th>Currency</th>
```

Comment:
I think we can mash together amount and currency in one column, meaning the column could be amount and then in the rows {{amount}} {{currency}}